.markdownlint.json

```
@@ -0,0 +1,8 @@
+{
+    "default": true,
+    "MD004": { "style": "asterisk"},
+    "MD013": false,
+    "MD026": { "punctuation": ".,;:!。，；：！" },
+    "MD033": false,
+    "MD041": false
+}
```

nit: Missing new line at EOF
```suggestion
}

```